### PR TITLE
Checklist Review

### PR DESCRIPTION
**Pull request by arcturus2**
### Description of the Change

The most important changes are in 28111aaf0955546f1622163abd2c8f2a5134c746 and the rest are formatting changes.

Also have a look at the section in "Unit testing changes in your PR" making note of a minimum of 80% code coverage required for new code.

This is a chance to also make a call on what should be part of the PR process going forward.

### CODE REVIEW

File: CHECKLIST.md [hunk 1]
```diff
@@ -3,51 +3,63 @@
...
+    - Use `Mockito`, `FxToolkit` and `FxRobot` as required.
+
+- [ ] Update the [change log](CHANGELOG.md) to inform developers of
+    important improvements or changes
 
     -   You must start with Added, Fixed, Improved, Moved, Removed,
         Renamed or Updated and sort them alphabetically
 
-- [ ] Update whatsnew.txt if you want to inform users about your change
+- [ ] Update `whatsnew.txt` if you want to inform users of your change
```
Comment (aldebaran30701) [inline CHECKLIST.md:39]: A link to whatsnew.txt would be nice, similar to changelog above. (for consistency)

**Review by aldebaran30701 — COMMENTED** [review]
Just a small thing

File: CHECKLIST.md (hunk 1 above)
Comment (arcturus2) [reply CHECKLIST.md]: No worries @aldebaran30701 that has been updated. Let me know if you can think of anything else.

**Review by antares1470 — CHANGES_REQUESTED** [review]

File: CHECKLIST.md [hunk 2]
```diff
@@ -3,51 +3,64 @@
...
 
-- [ ] Utility methods should end with the word Utilities (i.e not Util
-    or Utility)
+- [ ] Unit testing changes in your PR
+
+    - Unit testing changes you have made is required and should be
+    over 80% code coverage as a minimum.
+
+    - If any classes you have changed do not already have a unit tests
+    then it is **NOT** apon you to test the entire class, rather the
```
[the remark this answers is not in the document]
Comment (arcturus2) [reply CHECKLIST.md]: Corrected thank you.

File: CHECKLIST.md [hunk 3]
```diff
@@ -3,51 +3,64 @@
...
 
-- [ ] Avoid OS dependant code (e.g. Windows or Linux)
+- [ ] Javadoc classes and methods
 
-- [ ] Utility methods should end with the word Utilities (i.e not Util
-    or Utility)
+- [ ] Unit testing changes in your PR
+
+    - Unit testing changes you have made is required and should be
+    over 80% code coverage as a minimum.
```
[the remark this answers is not in the document]
Comment (arcturus2) [reply CHECKLIST.md]: Updated

**Review by aldebaran30701 — APPROVED** [review]

**Review by antares1470 — APPROVED** [review]

**Review by Atlas139mkm — APPROVED** [review]

**Review by sol695510 — APPROVED** [review]

**Review by Auriga2 — APPROVED** [review]
